project2/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <title>Movie Card</title>
+</head>
+
+<body>
+    <div class="card">
+        <div class="transbox">
+            <h2>THE GRAND BUDAPEST HOTEL</h2>
+            <p class="date">2014 | 1h 39min | IMDb 8.5 / 10</p>
+            <p>The adventures of Gustave H,
+                a legendary concierge at a famous hotel
+                from the fictional Republic of Zubrowka between
+                the first and second World Wars, and Zero Moustafa,
+                the lobby boy who becomes his most trusted friend.</p>
+            <div class="playbox">
+                <a href="#">PLAY</a>
```

Comment:
Same note about ALL CAPS here:
```suggestion
                <a href="#">Play</a>
```